app/services/credits/applied_coupon_service.rb

```diff
@@ -2,9 +2,10 @@
 
 module Credits
   class AppliedCouponService < BaseService
-    def initialize(invoice:, applied_coupon:)
+    def initialize(invoice:, applied_coupon:, base_amount_cents:)
       @invoice = invoice
       @applied_coupon = applied_coupon
+      @base_amount_cents = base_amount_cents
```

Comment:
What do you think of adding a comment to explain what is this `base_amount_cents`?

Reply:
added:)